spec/system/adjustment_system_spec.rb

```diff
@@ -21,6 +21,7 @@
       end
 
       it "allows you to choose items that do not yet exist" do
+        find('select option[data-select2-id="3"]', wait: 10)
```

Comment:
Where does the "3" come from? Is it the ID of a particular object? If so, make sure you're passing the record's actual ID in, or this will still be flaky if that ID changes (e.g. if the tests are run in a different order).

Reply:
I can answer this question since I made the suggestion. 

select2 uses `data-select2-id` as a unique identifier. it defaults to 2 and every time you make a selection it increments the id. so 3 is because default state was 2 => made a selection => got incremented => 3

you could make potentially make it check that the page had the initial value and then it changed instead:
```rb
expect(page).to have_css('select option[data-select2-id="2"]')
select @storage_location.name, from: "From storage location"
expect(page).to_not have_css('select option[data-select2-id="2"]', wait: 10)
```

Reply:
This makes me uncomfortable since it looks like this ID number depends on the number of select2's on the page and their state. If we change something on the page, this will break.

I'd rather have another way to get to this `select` (e.g. through a parent or sibling selector).